libs/util/misc/src/boolean/boolNullable.ts

```
@@ -0,0 +1,16 @@
+export const boolNullable = (value?: unknown): boolean | null => {
+  switch (typeof value) {
+    case 'boolean':
+      return value;
+    case 'object':
+    case 'bigint':
+    case 'number':
+    case 'string':
+      return !!value;
+    case 'undefined':
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)